docs/README.md

```diff
@@ -34,13 +34,6 @@ or use the appropriate install script:
 > but they can't be used in all cases. See [Limitations](./using-the-nuget-packages.md#limitations)
 > for details.
 
-To see the telemetry from your application directly on the standard output, set
-the following environment variables to `true` before launching your application:
-
-- `OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED`
-- `OTEL_DOTNET_AUTO_METRICS_CONSOLE_EXPORTER_ENABLED`
-- `OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED`
-
```

Comment:
Could you please describe the new approach here?
Debugging purposes seems to be important feature at the beginning of the journey with OTel .NET Auto.

Reply:
fixed in 984a940f0808cf6b22c83ef6258836f9eed2549a